demisto_sdk/commands/common/tests/integration_test.py

```diff
@@ -2179,6 +2179,25 @@ def test_additional_info_contained(self):
             {"name": "feedReputation", "required": "false", "hidden": True}
         ]
     }
+    HIDDEN_TRUE_BUT_REPLACED_USER = {
+        "configuration": [
+            {"type": 4, "display": "Username", "hidden": True},
+            {"type": 9, "display": "Username"},
+        ]
+    }
+    HIDDEN_TRUE_BUT_REPLACED_PASSWORD = {
+        "configuration": [
+            {"type": 4, "display": "Api key", "hidden": True},
+            {"type": 9, "displaypassword": "API key"},
+        ]
+    }
+    HIDDEN_TRUE_BUT_REPLACED_AND_NOT_REPLACED = {
```

Comment:
`AND_NOT_REPLACED`?

Reply:
Two parameters with type 4, one replaced and one not, so false should be returned.

Reply:
let's use `HIDDEN_TRUE_BUT_NOT_REPLACED_TYPE_4_TO_9`